bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceAction.java

```diff
@@ -54,6 +58,36 @@
  */
 public class FindReplaceAction extends ResourceAction implements IUpdate {
 
+	private static final String INSTANCE_SCOPE_NODE_NAME = "org.eclipse.ui.editors"; //$NON-NLS-1$
+
+	private static final String USE_FIND_REPLACE_OVERLAY = "useFindReplaceOverlay"; //$NON-NLS-1$
+
+	private static final String FIND_REPLACE_OVERLAY_AT_BOTTOM = "findReplaceOverlayAtBottom"; //$NON-NLS-1$
+
+	private boolean shouldUseOverlay() {
+		IEclipsePreferences preferences = InstanceScope.INSTANCE.getNode(INSTANCE_SCOPE_NODE_NAME);
+		boolean overlayPreference = preferences.getBoolean(USE_FIND_REPLACE_OVERLAY, true);
```

Comment:
```suggestion
		boolean overlayPreference = preferences.getBoolean(USE_FIND_REPLACE_OVERLAY, false);
```
Before merging this, this needs to be changed to default to false.

Reply:
https://github.com/eclipse-platform/eclipse.platform.ui/commit/67ef99dd0640b16d57511fe574efb5974ee221e9